dev-lang/clips/clips-6.3.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="C Language Integrated Production System"
+DESCRIPTION="
```

Comment:
Keep DESCRIPTION on one line according to the guidelines https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description